Sources/DittoPermissionsHealth/PermissionsHealth.swift

```diff
@@ -0,0 +1,150 @@
+                            Divider()
+                            Button(action: {
+                                openBluetoothSettings()
+                            }) {
+                                Text("Enable Bluetooth")
+                                    .foregroundColor(.white)
+                                    .padding(7)
+                                    .background(Color.blue)
+                                    .cornerRadius(10)
+                            }
+                        }
+                    }
+                    .padding()
+                    .background(Color.white)
+                    .cornerRadius(10) // Adjust as needed for the roundness of the corners
+                    .shadow(radius: 5) // Adjust the radius for the intensity of the shadow
+                }
+                .padding()
+                
+                ZStack {
+                    VStack(alignment: .leading) {
+                        HStack {
+                            Text("Wi-fi Status")
+                                .font(.title)
+                                .fontWeight(.bold)
+                            
+                            Spacer()
+                            Image(systemName: networkManager.isWifiEnabled ? "checkmark.circle.fill" : "exclamationmark.triangle.fill")
+                                .foregroundColor(networkManager.isWifiEnabled ? .green : .gray)
+                                .font(.title)
+                        }
+                        Divider()
+                        Text("Wi-Fi: \(networkManager.isWifiEnabled ? "Enabled" : "Disabled")")
+                        if !networkManager.isWifiEnabled {
+                            Divider()
+                            Button(action: {
+                                openBluetoothSettings()
+                            }) {
+                                Text("Enable Wi-fi")
+                                    .foregroundColor(.white)
+                                    .padding(7)
+                                    .background(Color.blue)
+                                    .cornerRadius(10)
+                            }
+                        }
+                    }
+                    .padding()
+                    .background(Color.white)
+                    .cornerRadius(10) // Adjust as needed for the roundness of the corners
+                    .shadow(radius: 5) // Adjust the radius for the intensity of the shadow
+                }
+                .padding()
+            }
+        }
+        .padding()
+        .onAppear {
+            if let infoDict = Bundle.main.infoDictionary {
+                bluetoothAlwaysUsageDescription = infoDict["NSBluetoothAlwaysUsageDescription"] as? String
+                bluetoothPeripheralUsageDescription = infoDict["NSBluetoothPeripheralUsageDescription"] as? String
+                localNetworkUsageDescription = infoDict["NSLocalNetworkUsageDescription"] as? String
```

Comment:
Does this accurately reflect the current state of permissions? I would guess that these are always populated if the application will at some point request these permissions but to get the current state you need to ask `CBManager` for Bluetooth:
https://developer.apple.com/documentation/corebluetooth/cbmanager/3377595-authorization?changes=latest_minor

`Bundle.main` can also resolve differently than expected for things in an SPM Package when used from outside of that package so this might have unexpected behavior when embedded in a different app (e.g. resolving to the SPM package bundle which doesn't have these)

Reply:
Yeah, that portion was initially checking if they had added the necessary keys to the info.plist. Which I guess is less important to check. 
Apple does not provide an API to check if the Local Network has been authorized for the app.
I can update the current implementation to separate the bluetooth check into checking the authorization for app and the overall status of bluetooth for the device.
The current implementation also checks if wifi is on or off.

Reply:
That makes sense and I know Apple generally doesn't give a lot of insight into some of these.

I'm all for checking the Bundle and giving a developer message like "App does not request XYZ required permisson..." but I'd want to test embedding this in a different app and how the `Bundle` resolves because we might be sandboxed away from accessing the containing application's Info.plist.